bin/meshroom_batch

```diff
@@ -64,6 +64,10 @@ parser.add_argument('--forceCompute', help='Compute in all cases even if already
 
 parser.add_argument('--submit', help='Submit on renderfarm instead of local computation.',
                     action='store_true')
+parser.add_argument("--submitLabel", 
+                    type=str, 
+                    default='%j [Meshroom]',
```

Comment:
Maybe use the same expression convention used in Meshroom (python format convention):
```suggestion
                    default='{projectName} [Meshroom]',
```

Reply:
Done.